src/expr/src/vector_op/position.rs

```diff
@@ -14,16 +14,43 @@
 
 use risingwave_expr_macro::function;
 
-use crate::Result;
-
-/// Location of specified substring
+/// Returns the index of the first occurrence of the specified substring in the input string,
+/// or zero if the substring is not present.
+///
+/// # Example
+///
+/// ```slt
+/// query I
+/// select position('om' in 'Thomas');
+/// ----
+/// 3
+///
+/// query I
+/// select strpos('hello, world', 'lo');
+/// ----
+/// 4
+///
+/// query I
+/// select strpos('high', 'ig');
+/// ----
+/// 2
+///
+/// query I
+/// select strpos('abc', 'def');
+/// ----
+/// 0
 ///
-/// Note: According to pgsql, position will return 0 rather -1 when substr is not in the target str
+/// query I
+/// select strpos('床前明月光', '月光');
```

Comment:
Does the  \`\`\`slt\`\`\` block mean that all these slt cases will be tested in e2e test?

Reply:
Yes. #5117